<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QwcConnectWidget</class>
 <widget class="QWidget" name="QwcConnectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect...</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="fContainer">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>117</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>99</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="application.qrc">:/icons/qwired_logo_48.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>12</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>&lt;b&gt;Welcome to Qwired!&lt;/b&gt;&lt;br&gt;&lt;br&gt;Please enter a server address and your login credentials, then press Connect. Alternatively you can select one of your bookmarks from the Bookmarks-Menu below.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>12</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Address:</string>
             </property>
             <property name="buddy">
              <cstring>fAddress</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="fAddress">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Login:</string>
             </property>
             <property name="buddy">
              <cstring>fLogin</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLineEdit" name="fLogin">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Pasword:</string>
             </property>
             <property name="buddy">
              <cstring>fPassword</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QLineEdit" name="fPassword">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QToolButton" name="fBtnFavourites">
             <property name="text">
              <string>Bookmarks...</string>
             </property>
             <property name="icon">
              <iconset resource="application.qrc">
               <normaloff>:/icons/icon_favourite.png</normaloff>:/icons/icon_favourite.png</iconset>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="btnConnect">
             <property name="text">
              <string>Connect</string>
             </property>
             <property name="icon">
              <iconset resource="application.qrc">
               <normaloff>:/icons/16x16/applications-internet.png</normaloff>:/icons/16x16/applications-internet.png</iconset>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>169</width>
             <height>98</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>116</width>
           <height>144</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>194</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="0">
          <widget class="QProgressBar" name="fProgress">
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="fBtnCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="fStatus">
           <property name="text">
            <string>Ready.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>194</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fAddress</tabstop>
  <tabstop>fLogin</tabstop>
  <tabstop>fPassword</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>fBtnFavourites</tabstop>
  <tabstop>fBtnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
